yaksh/templates/yaksh/courses.html

```diff
@@ -151,9 +151,14 @@
                     {% for unit in module.get_learning_units %}
                         <ul class="inputs-list">
                         <li>
-                        {% if unit.type == "quiz" %}
-                            <a href="{{URL_ROOT}}/exam/manage/addquiz/{{unit.quiz.id}}/{{course.id}}">
-                            {{unit.quiz.description}}</a>
+                        {% if unit.learning_type == "quiz" %}
```

Comment:
This should `type` instead of `learning_type`

Reply:
Wasn't this supposed to be changed to ```category``` ?

Reply:
I changed this to `type`. I feel it is ok.

Reply:
done